app/scripts/modules/amazon/src/instance/amazon.instance.write.service.ts

```diff
@@ -64,25 +54,25 @@ export class AmazonInstanceWriter extends InstanceWriter {
     });
   }
 
-  public registerInstancesWithTargetGroup(
+  public static registerInstancesWithTargetGroup(
     instanceGroups: IMultiInstanceGroup[],
     application: Application,
     targetGroupNames: string[],
   ) {
-    const jobs = this.buildMultiInstanceJob(
+    const jobs = super.buildMultiInstanceJob(
```

Comment:
TIL that typescript allows inheritance on static members

Reply:
not just typescript, es6 classes support this!